### PR TITLE
Page on Global.health

### PR DESCRIPTION
**Pull request by iamleeg**

### CODE REVIEW

File: content/projects/2020_global_dot_health.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+description: "Delivering a scalable platform for "
+date: 2020-09-18T16:00:00+01:00
+draft: false
+---
+
+## Global.health: a Data Science Initiative
+
+Researchers at the Boston Children's Hospital and collaborating institutions created [Healthmap](https://healthmap.org/), a tool for tracking the spread of epidemics in humans and animals. First used to great success in the Ebola outbreak of 2014, associating cases of "a mysterious haemmorhagic fever" with confirmed cases of Ebola to allow experts and the public to see a more accurate state of the spread of the virus, Healthmap continued to be applied through the SARS and MERS epidemics, and is regularly used to track seasonal flu.
+
+The rapid global spread of the COVID-19 pandemic overwhelmed both the manual processes used by the Healthmap team, and the digital infrastructure used to record the data. A collaboration between BCH, Oxford University, and Google.org led to Global.health: an improved, scalable platform for disease tracking. Real-time acquisition of case data shows where potential hotspots may break out, and the effect of policy changes on the spread of the virus.
```
Comment (martinjrobins) [inline content/projects/2020_global_dot_health.md:13]: is there a link to Global.health, or at least a link to more info about it?

**Review by martinjrobins — CHANGES_REQUESTED** [review]
looks great @iamleeg, might want to put in a link to Global.health?

File: content/projects/2020_global_dot_health.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+title: "Global.health"
+client: "[Global.health: a Data Science Initiative](https://global.health/)"
+description: "Delivering a scalable platform for "
```
Comment (abhidg) [inline content/projects/2020_global_dot_health.md:4]: scalable platform from global health?

File: content/projects/2020_global_dot_health.md (hunk 1 above)
Comment (iamleeg) [reply content/projects/2020_global_dot_health.md]: Yes, I added the links.

File: content/projects/2020_global_dot_health.md (hunk 2 above)
Comment (iamleeg) [reply content/projects/2020_global_dot_health.md]: Oops :)

**Pull request merged**